### ISSUE TITLE
Is 3ML ready for a Python 3.9 environment?

### ISSUE THREAD

**Issue by eblur** [threeML/threeML#530]
**Describe the bug**

Conda install of 3ML does not allow for Python 3.9

**To Reproduce**

Here is my log from an attempt to install 3ML with Conda.

```
liac-mbp:multiwave liac$ conda install -c conda-forge -c threeml astromodels threeml
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: - 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                  
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - threeml -> python[version='>=2.7,<2.8.0a0|>=3.7,<3.8.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
```

**Desktop (please complete the following information):**
 - OS: Mac Big Sur 12.1
 - Python 3.9

**Additional context**

I prefer not to downgrade my core Python version. I am already using a separate conda environment and want to use the latest versions of Astropy, Numpy, etc in conjunction with 3ML.

**Comment by grburgess** [comment]
We need to bump the conda version for sure. @ndilalla you want to grab this? Or I can.

**Comment by ndilalla** [comment]
Hi @eblur, 3ML on conda is released only for python 3.7 at the moment. The easiest way to install 3ML and astromodels with python 3.9 is by using pip on your conda environment:
`pip install threeML`
`pip install astromodels`

**Comment by omodei** [comment]
We are now compatible with python 3.9
